packages/app-config/scripts/package-subtree-local.sh

```
@@ -0,0 +1,1094 @@
+      awk -v file="$rel_file" '
+        /^diff --git/ {
+          in_file = ($0 ~ " b/" file "$" || $0 ~ " b/" file " ")
+          if (in_file) print
+          next
+        }
+        in_file { print }
+        /^diff --git/ && !in_file { exit }
+      ' "$patch_file" > "$temp_patch"
+
+      if [ -s "$temp_patch" ]; then
+        cat >> "$target_file" << EOF
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)